### DISCUSSION TITLE
Create bridge device on GUI.

### DISCUSSION THREAD

**Discussion by gilbertoferreira** [linuxdeepin/developer-center#3326] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hello there.
Often I need create some bridge in Linux in order do work fine with Virtmanager for instance.
But when using Deepin Linux I can't create a bridge on GUI.
Here's an example in KDE Plasma.
![Screenshot_20220724_221922](https://user-images.githubusercontent.com/10462980/180675597-10993734-8a1f-4f98-977a-b39bcb300649.png)

**Comment by BLumia** [comment]
@meiyixiang 需求：希望能通过设置界面创建（网络）桥接设备。
